src/main/java/com/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,9 +1,16 @@
 package com.bookshop.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.PositiveOrZero;
 import java.math.BigDecimal;
+import org.hibernate.validator.constraints.ISBN;
 
-public record CreateBookRequestDto(String title, String author,
-                                  String isbn, BigDecimal price,
-                                  String description, String coverImage) {
+public record CreateBookRequestDto(@NotBlank String title,
+                                   @NotBlank String author,
+                                   @NotBlank @ISBN(type = ISBN.Type.ANY) String isbn,

```

Comment:
also you can create your own `@Isbn` annotation or add `@Pattern` to validate isbn in correct format